From bzolnier@gmail.com  Tue Aug 25 22:18:57 2009
From: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Date: Sun, 23 Aug 2009 15:29:19 +0200
Subject: Staging: rt3070: update allowed channels list for Japan
To: "Greg Kroah-Hartman" <gregkh@suse.de>
Cc: devel@driverdev.osuosl.org, Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>, Marcin Slusarz <marcin.slusarz@gmail.com>, Mike Galbraith <efault@gmx.de>, linux-kernel@vger.kernel.org
Message-ID: <20090823132919.17784.65031.sendpatchset@localhost.localdomain>


From: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>

rt3070 driver allows use of 5G channel 34 while rt{286,287,309}0
drivers don't and quick googling seems to confirm the limitation.

Signed-off-by: Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rt2860/chlist.h |    5 -----
 1 file changed, 5 deletions(-)

--- a/drivers/staging/rt2860/chlist.h
+++ b/drivers/staging/rt2860/chlist.h
@@ -524,12 +524,7 @@ static CH_REGION ChRegion[] =
 			JAP,
 			{
 				{ 1,   14,  20, BOTH, FALSE},	// 2.4 G, ch 1~14
-#ifndef RT30xx
 				{ 36, 	4,  23, IDOR, FALSE},	// 5G, ch 36~48
-#endif
-#ifdef RT30xx
-				{ 34, 	4,  23, IDOR, FALSE},	// 5G, ch 34~46
-#endif
 				{ 0},							// end
 			}
 		},
